scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -21,14 +21,16 @@ This notebook will demonstrate how to:
 ---
 
 In this notebook, we'll perform integration on scRNA-seq datasets from the [Single-cell Pediatric Cancer Atlas (`ScPCA`)](https://scpca.alexslemonade.org/), a database of uniformly-processed pediatric scRNA-seq data built and maintained by the Data Lab.
-This database currently hosts (as of March 2023) eleven projects from eight different ALSF-funded labs who have generated single-cell pediatric cancer transcriptomic data with the ultimate goal of making this data easily accessible to investigators (like you!).
-`ScPCA` raw data is processed with [nextflow pipeline](https://github.com/AlexsLemonade/scpca-nf) that uses [`alevin-fry`](https://doi.org/10.1038/s41592-022-01408-3) for fast and efficient mapping, followed by filtering and normalization with Bioconductor tools. 
+The `ScPCA` database currently hosts single-cell pediatric cancer transcriptomic data generated by ALSF-funded labs, with a leading goal of making this data easily accessible to investigators (like you!).
```

Comment:
I don't love "leading goal"... "primary goal", "main goal"? None of those sound quite right. I have "the" goal below, because just "a goal" sounded bad. 
```suggestion
The `ScPCA` database currently hosts single-cell pediatric cancer transcriptomic data generated by ALSF-funded labs, with the goal of making this data easily accessible to investigators (like you!).
```